packages/anon-aadhaar-pcd/src/utils.ts

```diff
@@ -261,3 +261,114 @@ export function decompressByteArray(byteArray: Uint8Array) {
   const decompressedArray = pako.inflate(byteArray)
   return decompressedArray
 }
+
+export const enum SELECTOR_ID {
+  emailOrPhone = 0,
+  referenceId,
+  name,
+  dob,
+  gender,
+  cafeof,
+  district,
+  landmark,
+  house,
+  location,
+  pinCode,
+  postOffice,
+  state,
+  stress,
+  subDistrict,
+  VTC,
+}
+
+export class SelectorBuilder {
+  selector: number[]
+
+  constructor() {
+    this.selector = new Array(16).fill(0)
+  }
+
+  selectEmailOrPhone() {
```

Comment:
I think this is not required. Email and Phone is not actually present in the data - this field indicates if the user has either email or phone verified with Aadhar - not sure if a verifier needs this info? 🤔